website/pages/style.css

```diff
@@ -12,7 +12,7 @@
 @import '~@hashicorp/react-consent-manager/style.css';
 @import '~@hashicorp/react-content/style.css';
 @import '~@hashicorp/react-docs-page/style.css';
-@import '~@hashicorp/react-docs-sidenav/style.css';
+
```

Comment:
This component now uses CSS modules, so a global style import is no longer needed.